CASSANDRA.md

```diff
@@ -16,7 +16,7 @@ All examples have been properly tested, and they should work out of the box.
 
 ## Prerequisites ##
 
-The keyspace and column family needed for all examples are defined in this CQL script:
+The keyspace and column family needed for all examples are defined in this CQL script. This step can be entirely skipped in Phantom were schema is auto-generated.
```

Comment:
@fwbrasil I think it's a very important distinction, I'll phrase it better, but unlike Quill Phantom expects to manage schema creation and migrations alike, to the best of my knowledge Quill relies on external pre-existing schemas.